storm-client/src/jvm/org/apache/storm/trident/spout/IPartitionedTridentSpout.java

```diff
@@ -36,7 +37,7 @@
     interface Coordinator<Partitions> {
         /**
          * Return the partitions currently in the source of data. The idea is is that if a new partition is added and a prior transaction is
-         * replayed, it doesn't emit tuples for the new partition because it knows what partitions were in that transaction.
+         * replayed, it doesn't emit tuples for the new partition because it knows what partitions were in that transaction.jobh
          */
```

Comment:
Sorry, what is the jobh mean?

Reply:
Whoops, yes. Please remove this.

Reply:
thanks for comments folks, I remove typos here and add more comments